### ISSUE TITLE
How to get native event object in button click event handler?

### ISSUE THREAD

**Issue by aztack** [cocopon/tweakpane#542]
```js
pane.addButton({title: 'Save'}).on('click', e => {
  // if ctrlKey is pressed
  ?
  // else
 ///
});
```

**Comment by aztack** [comment]
I found the solution:
```js
pane.addButton({title:'Save'}).element.on('click', e => {
 //...
});
```

**Comment by cocopon** [comment]
Implemented in #591. You'll be able to access the native event object via `e.native` from the next version.

```js
pane.addButton({title: 'Save'}).on('click', (ev) => {
  console.log(ev.native);
});
```

**Comment by cocopon** [comment]
Released 4.0.3.
